src/widget-core/README.md

```diff
@@ -557,6 +557,85 @@ render() {
 }
 ```
 
+#### Managing Themes Throughout an Application
+
+To manage themes throughout an application, the `ThemedMixin` leverages the [Injectors, Providers & Containers](#injectors-providers--containers) to set an application's locale and inject the locale data into all widgets using the `ThemedMixin`. When the locale is updated in the theme `Injector` all themed widgets will be invalidated and re-rendered with the updated locale.
```

Comment:
This mentions locales rather than themes near the end

Reply:
copy paste special, thanks @matt-gadd

Reply:
FYI, think this is still an issue in the PR's current readme.

Similar to i18n, I'm busy porting this over (with s/locale/theme/) to the theme ref guide so this section will shortly be redundant anyway.